src/common/fastopt.cpp

```diff
@@ -94,6 +94,7 @@ template struct As<std::vector<int>>;
 // Windows: size_t = unsigned long long (8 bytes), uint64_t = unsigned long long (8 bytes)
 template struct As<std::vector<unsigned long long>>;
 template struct As<std::vector<unsigned long>>;
+template struct As<std::vector<unsigned int>>;
```

Comment:
@XapaJIaMnu What is the reason of adding this for `int` data type? I believe your code change must be using it but I am trying to understand where 👍

Reply:
So, there is a call of `As<std::vector<size_t>>` which is in `fastopt.h`. Depending on the architecture (and the platform) `size_t` resolves to a different thing. When compiling to wasm, I think it resolves to `unsigned long`, but when compiling with gcc `-m32` it resolves to `unsigned int` which is not defined in the template definition (even if they have the same  width)..

Reply:
Thanks for the explanation 👍